pages/common/git-ls-tree.md

```diff
@@ -0,0 +1,12 @@
+# git ls-tree
+
+> List the contents of a tree object.
+> More information: <https://git-scm.com/docs/git-ls-tree>.
+
+- List the contents of the tree on a branch:
+
+`git ls-tree {{master}}`
+
+- List the contents of the tree on a commit, recurse into subtrees, and show only filenames:
+
+`git ls-tree -r --name-only {{c39ab84d}}`
```

Comment:
I'd prefer using a descriptive name here, like `{{commit_hash}}`.

Also, we usually try to avoid introducing too many new options in a single example, to make sure the pages remain beginner-friendly.

I would suggest splitting this example into two: one for the `-r` option, and one for the `--name-only` option (maybe use a different object for the third example, e.g. a tag or a relative reference like `HEAD~3`.

Reply:
Sure ;)

Done in 35035345650e07534a454774f0aa987ca60c6124